mods/flowers/init.lua

```diff
@@ -46,6 +46,7 @@ local function add_simple_flower(name, desc, box, f_groups)
 		paramtype = "light",
 		walkable = false,
 		buildable_to = true,
+		stack_max = 121,
```

Comment:
:-1: 99 was used before but we removed that.
The realism of carrying objects in MT is irrelevant here, we carry 64 cubic metres of stone too.

Reply:
I've removed this change.

Reply:
Thanks.